website/docs/r/namespace.html.md

```diff
@@ -29,3 +29,36 @@ The following arguments are supported:
 ## Attributes Reference
 
 * `id` - ID of the namespace.
+
+## Import
+
+Namespaces can be imported using its `name` as accessor id, e.g.
+
+```
+$ terraform import vault_namespace.example <name>
+```
+
+If the declared resource to be imported is intended to support namespaces using a [provider alias](https://registry.terraform.io/providers/hashicorp/vault/latest/docs#using-provider-aliases) then the <name> is relative to the namespace path, e.g.
```

Comment:
Here's a suggestion:
If the declared resource is imported and intends to support namespaces using a provider alias, then the <name> is relative to the namespace path.